providers/src/airflow/providers/dbt/cloud/hooks/dbt.py

```diff
@@ -356,14 +356,23 @@ def get_account(self, account_id: int | None = None) -> Response:
         return self._run_and_get_response(endpoint=f"{account_id}/")
 
     @fallback_to_default_account
-    def list_projects(self, account_id: int | None = None) -> list[Response]:
+    def list_projects(
+        self, name_contains: str | None = None, account_id: int | None = None
```

Comment:
```suggestion
        self, account_id: int | None = None, name_contains: str | None = None
```
changing order might break existing code

Reply:
+1 I've seen folks get burned by this before.